### DISCUSSION TITLE
Disable registration when installing wallabag?

### DISCUSSION THREAD

**Discussion by nicosomb** [wallabag/wallabag#6785] [Ideas]
I was browsing through the issues and I saw some logs sent by users. 

In these logs, there are sometimes wallabag instances URL. 

Most of these instances have the "Register" button which is enabled. This setting can be disabled (see this file https://github.com/wallabag/wallabag/blob/master/app/config/parameters.yml.dist#L40). 

But I think that it should be a manual action: when installing wallabag, it could better to decide if we want to open registration. Not if we want to disable it. 

It can avoid surprises (and spam) for small instances. 

Before opening a (small and easy) pull request, I prefer have your opinion.

**Comment by Kdecherf** [comment]
Indeed, making this an opt-in option looks better

**Comment by nicosomb** [comment]
I just opened two pull requests: https://github.com/wallabag/wallabag/pull/6791 & https://github.com/wallabag/docker/pull/357

**Comment by nicosomb** [comment]
Will be fixed in 2.7.0.
